apps/src/code-studio/components/SmallFooter.jsx

```diff
@@ -269,11 +268,17 @@ export default class SmallFooter extends React.Component {
               )}
             />
             <Button
+              aria-label={i18n.closeDialog()}
+              icon={{
+                iconName: 'xmark',
+                iconStyle: 'light',
+              }}
               id="x-close-copyright"
+              isIconOnly
               onClick={() => this.setState({menuState: MenuState.MINIMIZED})}
-              icon="fa-solid fa-xmark"
+              size="l"
               style={styles.copyrightXClose}
-              aria-label={i18n.closeDialog()}
+              type="primary"
```

Comment:
I did have to update this component so it used the new Design component, but the copyright area will be moved to a dialogue window in a followup PR.